Assignment2/SWAPI.yaml

```diff
@@ -0,0 +1,346 @@
+swagger: '2.0'
+info:
+  description: >-
+    The Star Wars API is the world's first quantified and
+    programmatically-formatted set of Star Wars data. After hours of watching
+    films and trawling through content online, we present to you all the People,
+    Films, Species, Starships, Vehicles and Planets from Star Wars.
+  version: ''
+  title: SWAPI-The Star Wars API
+host: swapi.co
+basePath: /api
+tags:
+  - name: People
+    description: >-
+      A People resource is an individual person or character within the Star
+      Wars universe.
+schemes:
+  - http
+paths:
+  /people/:
+    get:
+      tags:
+        - People
+      summary: get all the people resources
+      description: Returns all characters in array
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: OK
+          schema:
+            $ref: '#/definitions/People'
+        '301':
+          description: Moved Permanently
+        '404':
+          description: Not found
+  '/people/{:ID}':
+    get:
+      tags:
+        - People
+      summary: get a specific people resource
+      description: Returns a single character
+      produces:
+        - application/json
+      parameters:
+        - name: ':ID'
+          in: path
+          description: ID of character to return
+          required: true
+          type: integer
+          format: int64
+      responses:
+        '200':
+          description: OK
+          schema:
+            $ref: '#/definitions/ID'
+        '301':
+          description: Moved Permanently
+        '404':
+          description: Not found
+  /people/schema:
+    get:
+      tags:
+        - Schema
+      summary: view the JSON schema for this resource
+      description: ''
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: OK
+          schema:
+            $ref: '#/definitions/Schema'
+        '301':
+          description: Moved Permanently
+        '404':
+          description: Not found
+definitions:
+  People:
+    properties:
+      count:
+        type: integer
+        example: 57
+      next:
+        type: string
+        example: 'http://swapi.co/api/people/?page=2'
+      previous:
+        type: string
+        example: null
+      result:
+        type: array
+        items:
+          type: string
+          $ref: '#/definitions/ID'
+  ID:
```

Comment:
what kind of Schema name is ID? name the schema according to what it represents!